### DISCUSSION TITLE
Intelisense and JDK_HOME not work with vscode in WSL2

### DISCUSSION THREAD

**Discussion by lokize** [jbangdev/jbang#1325] [Q&A] [answered]
Hello friends, I'm currently using WSL 2 and my JDK is installed and configured with jbang in the wsl environment (centos), but the Intelisense of java and jdk is not found when I use vscode on windows, is there any way to fix this problem without having to install jdk on windows?

i trying this but not working:
![image](https://user-images.githubusercontent.com/19701630/161448277-d572c2dd-c229-4559-b98d-58b9e511a100.png)

**Comment by maxandersen** [comment, marked as answer]
What editor are you using ? Vscode ? I haven't personally tried how it works but my assumption is that if you are running vscode in windows and not under wsl you would need a java installed under windows too.

**Reply by lokize** [reply]
I thought there was some other alternative, anyway thanks a lot!
